model/flow/ledger.go

```diff
@@ -78,6 +78,13 @@ func CadenceRegisterID(owner []byte, key []byte) RegisterID {
 	}
 }
 
+func NewRegisterID(owner, key string) RegisterID {
+	return RegisterID{
+		Owner: addressToOwner(BytesToAddress([]byte(owner))),
```

Comment:
```suggestion
		// left-pad owner with zeros to ensure owner has 8 bytes.
		Owner: addressToOwner(BytesToAddress([]byte(owner))),
```

Reply:
i'll keep this as is for now.  this conversion won't be needed in the future (i'm planning to replace Owner's type from string to Address)